SXR/SDK/sxrsdk/src/main/jni/objects/vertex_buffer.cpp

```diff
@@ -390,6 +392,52 @@ namespace sxr {
         return 0;
     }
 
+    void VertexBuffer::transform(glm::mat4& mtx, bool doNormals)
+    {
+        const DataEntry* normEntry = find("a_normal");
+        float*    data = reinterpret_cast<float*>(mVertexData);
+        int       stride = getVertexSize();
+
+        if (data == NULL)
+        {
+            return;
+        }
+        markDirty();
```

Comment:
Shouldn't markDirty be called while holding mLock?

Reply:
yes i will fix it